tchannel/tornado/dispatch.py

```diff
@@ -182,9 +185,11 @@ def handle_call(self, request, connection):
                 )
 
                 # get new top-level resp from controller
-                new_resp = yield gen.maybe_future(
-                    handler.endpoint(new_req)
-                )
+                with StackContext(lambda: _context):
+                    tchannel.get_context().parent_tracing = request.tracing
+                    f = handler.endpoint(new_req)
+
```

Comment:
Yes, I know. `parent_tracing` is tracked by context. It is part of the state that it tracks per 'async thread'. I'm assuming `Context` is the class that will hold all this state. Right now, you'll do 

    with Context(parent_tracing):
      # ...

But if we decide to add another piece of state to be tracked, you'd basically change it to

    with Context(parent_tracing, second_state):
        # ...